rfcs/0003-kubernetes-oci/README.md

```diff
@@ -47,25 +47,26 @@ and push the archive to a container registry as an OCI artifact.
 ```sh
 flux push artifact oci://docker.io/org/app-config:v1.0.0 \
   --source="$(git config --get remote.origin.url)" \
-  --revision="$(git branch --show-current)/$(git rev-parse HEAD)" \
+  --revision="$(git rev-parse HEAD)" \
   --path="./deploy"
 ```
 
-The Flux CLI will produce artifacts of type `application/vnd.docker.distribution.manifest.v2+json`
-which ensures compatibility with container registries that don't support custom OCI media types.
+The Flux CLI will produce OCI artifacts by setting the config layer
+media type to `application/vnd.cncf.flux.config.v1+json`.
 
 The directory pointed to by `--path` is archived and compressed in the `tar+gzip` format
-and the layer media type is set to `application/vnd.docker.image.rootfs.diff.tar.gzip`.
+and the layer media type is set to `application/vnd.cncf.flux.content.v1.tar+gzip`.
 
-The source URL and revision are added to the OCI artifact as annotations in the format:
+The source and revision are added to the OCI artifact as Open Containers standard annotations:
 
 ```json
 {
   "schemaVersion": 2,
-  "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
```

Comment:
Is it new? It feels to me like this is already implemented.

If not, these changes (`manifest mediaType` and `annotations` keys) are quite important for users relying on it. It should at least be in a different commit.

Reply:
The annotations are not new, the media type changed to OCI in this proposal.